packages/qwik/src/core/client/vnode-namespace.ts

```diff
@@ -293,3 +300,22 @@ interface NewElementNamespaceData {
   elementNamespace: string;
   elementNamespaceFlag: number;
 }
+
+export function getAttributeNamespace(attributeName: string): string | null {
+  switch (attributeName) {
+    case 'xlink:href':
+      return XLINK_NS;
+    case 'xlink:actuate':
+    case 'xlink:arcrole':
+    case 'xlink:role':
+    case 'xlink:show':
+    case 'xlink:title':
+    case 'xlink:type':
+    case 'xml:base':
+    case 'xml:lang':
+    case 'xml:space':
+      return XML_NS;
+    default:
+      return null;
+  }
+}
```

Comment:
we should benchmark this vs 2 regexes, but this seems to indicate a prepared regex is way faster: https://jsben.ch/XSViz